pyproject.toml

```diff
@@ -26,11 +26,11 @@ tlv8 = "^0.10"
 
 # lpc55
 crcmod = "^1.7"
-cryptography = ">=42"
+cryptography = ">=41"
 hidapi = "^0.14"
 
 # nrf52
-ecdsa = "^0.19"
+ecdsa = ">=0.18"
```

Comment:
Did you intentionally remove the upper boundary?  I’d like to keep it as `ecdsa` does not have a documented versioning policy.  (For `cryptography` it’s fine as they guarantee a stable public API.)